### ISSUE TITLE
Let the user set the location of dlp to call

### ISSUE THREAD

**Issue by realjordanna** [GoogleCloudPlatform/dlp-dataflow-deidentification#85]
Parent is set only with a project id meaning all calls go to "global" instead of specific regions.

**Comment by sasirekhamsvl** [comment]
yes, I too need the current git repo supports regional too instead of "global". The organization I am working only allows "asia-south1".
I can't actually create the de-identify templates json files because it is throwing 403 error to me. Could you modify the yaml files to ask for regions
